dbt/adapters/default/impl.py

```diff
@@ -823,14 +819,15 @@ def run_operation(cls, profile, project_cfg, manifest, operation_name,
     @classmethod
     def get_catalog(cls, profile, project_cfg, manifest):
         try:
-            results = cls.run_operation(profile, project_cfg, manifest,
-                                        GET_CATALOG_OPERATION_NAME,
-                                        GET_CATALOG_RESULT_KEY)
+            table = cls.run_operation(profile, project_cfg, manifest,
+                                      GET_CATALOG_OPERATION_NAME)
         finally:
             cls.release_connection(profile, GET_CATALOG_OPERATION_NAME)
+
         schemas = list({
             node.to_dict()['schema']
             for node in manifest.nodes.values()
         })
-        results = results.table.where(lambda r: r['table_schema'] in schemas)
+
+        results = table.where(lambda r: r['table_schema'] in schemas)
```

Comment:
I think I found a bug that previously existed here -- if the value of `r['table_schema']` is uppercased, but all of the `schemas` grabbed from the manifest are lowercased, then `results` will be empty.

This would happen if the schema name in the target is eg. `analytics` and quoting is turned off on Snowflake. Can we make this lookup case-insensitive?

Reply:
OR: I think we have some logic to case-insensitive match up catalog entries to the manifest over in `task/generate.py`. Do we even need this logic still?

Reply:
We can definitely make this lookup case-insensitive, and I think that's the cleanest path. I don't see anything in `task/generate.py` that does a case-insensitive match either.

Reply:
@beckjake https://github.com/fishtown-analytics/dbt/blob/development/dbt/contracts/graph/manifest.py#L300-L302

Reply:
Oh! but that's a bit different since we don't really care about the table names, right? I think it's probably better to just call `.lower()` on `node.to_dict()['schema']` above and then have the lambda call `.lower()` on its input instead of trying to make that method handle both cases or anything.

Reply:
oh, good point @beckjake. Yeah, i think just lowercasing both things here makes sense to me